### DISCUSSION TITLE
Unable to compile llama.cpp with CUDA support via `npx --no node-llama-cpp download`

### DISCUSSION THREAD

**Discussion by AlureonTheVirus** [withcatai/node-llama-cpp#308] [General]
![image](https://github.com/user-attachments/assets/dfab1b52-353e-41cd-bb54-b9453d50f56a)
I have CUDA installed but for some reasons the script is hitting compilation errors. the log shows it seems to find my current installation but then I immediately get a CMake error stating `No CUDA toolset found.`

**Comment by giladgd** [comment]
Try to use the latest version of `node-llama-cpp` that I've just released (3.0.1), which includes prebuilt binaries for CUDA.
If it still doesn't work, see the updated [CUDA guide](https://node-llama-cpp.withcat.ai/guide/CUDA) for troubleshooting.

**Reply by AlureonTheVirus** [reply]
Awesome thanks! I'll validate tonight.
